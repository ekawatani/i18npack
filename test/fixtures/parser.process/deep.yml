foo: this
bar: "{{ foo }} and that"
baz: "{{ foo }} and {{ bar }}"