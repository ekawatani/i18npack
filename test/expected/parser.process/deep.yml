foo: this
bar: this and that
baz: this and this and that